src/helpers/utilityFunctions.ts

```diff
@@ -0,0 +1,56 @@
+import { ICard } from "../interfaces/card";
+import constants from './constants';
+import { sha256 } from 'js-sha256';
+import { Request } from 'express';
+import QRCode from 'qrcode';
+
+
+function initializePagination(req: Request): Object {
```

Comment:
Need to define `Object` with actual interface

Reply:
Oh yes, Objects are also typed I just updated it.